ui/app/AppLayouts/Communities/panels/SharedAddressesPanel.qml

```diff
@@ -226,6 +257,18 @@ Control {
             getCurrencyAmount: function (balance, symbol) {
                 return root.getCurrencyAmount(balance, symbol)
             }
+
+            Component.onCompleted: {
+                const tmpTokenCountMap = new Map()
```

Comment:
This make the flow really intertwined but I understand that's workaround for now... Added note to https://github.com/status-im/status-desktop/issues/14140 in order to refactor in the future.